majorkirby/majorkirby.py

```diff
@@ -298,7 +289,7 @@ def suffix(self):
         Returns:
           str: Suffix for stack name
         """
-        h = hashlib.sha256(json.dumps(self.get_raw_tags()))
+        h = hashlib.sha256(json.dumps(self.get_raw_tags()).encode("utf-8"))
```

Comment:
This was necessary to ensure UTF-8 strings are emitted for comparison (like, inside tests).